src/components/molecules/AnnouncementMessage/AnnouncementMessage.scss

```diff
@@ -7,7 +7,7 @@
   background: $primary;
   top: 66px;
   // NOTE: Currently, the sidebar takes 20% of the screen, thus 40% is half of map's occupation
-  left: calc((100% - #{$map-sidebar-width}) / 2);
+  left: 50%; // calc((100% - #{$map-sidebar-width}) / 2);
```

Comment:
@margulies can you please explain what the intent of this change is?

My understanding is that we did this to ensure that the sidebar is centered with the regards to the map (though my guess is this may have changed recently with some of @mike-lvov's chat refactor work in https://github.com/sparkletown/sparkle/pull/1179 related followups)

Reply:
If that's the case, and we no longer need to take the width into account here, then we should also cleanup the commented bit of code here too.

Reply:
The issue this is addressing is that with the refactored chat (#1179), the announcement banner now shifts its position depending on whether the chat sidebar is open or closed. I found the user experience better to keep the banner in a stable position. 

If that sounds good, just let me know and I'll clean up the comment.

Reply:
@0xdevalias it's more a question to @mcflyok if we need to center the sidebar, based on the chat's sidebar opened\closed state

Reply:
@mike-lvov @0xdevalias @margulies Sounds good to make the announcement banner centered counting the presence of the chat sidebar ! Nice one !

Reply:
@mcflyok I think what @margulies is saying is that he found it felt better when the announcement bar was stable, not moving with the chat sidebar.

While I haven’t played with both options, at least in its current state, I think I would tend to agree. The chat sidebar is currently a ‘slide out draw’ that sits above/outside the ‘flow’ of everything else. Therefore for it to ‘push’ the announcement banner when it opens would feel really weird to me.

IMO the sidebar should ‘push’ everything (map included), or push nothing (which is kind of how it is being proposed/implemented in this PR currently). My personal vote is for the latter.

Reply:
@0xdevalias @margulies ooops sorry guys didnt get it, I'm down to keep it stable and always centered as it is kinda part of the topbar. sorry for the misunderstanding

Reply:
No worries. Thanks for your input! :)